csharp/Facebook.Yoga/ios/CustomBuildAction.targets

```diff
@@ -0,0 +1,11 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<PropertyGroup>
+	    <CompileDependsOn>CopyInNativeLib;$(CompileDependsOn)</CompileDependsOn>
+	</PropertyGroup>
+
+	<!-- The # and , in this path does not play nice with the binding project logic, so make a copy -->
+	<Target Name="CopyInNativeLib" Inputs="..\..\..\buck-out\gen\yoga#default,static\libyoga.a" Outputs="$(ProjectDir)libyoga.a">
```

Comment:
With #286, we can use the fat static library `buck-out/gen/csharp/yoganet-ios/libyoga.a` that includes iOS simulator 64bit, arm64 and armv7.